File: 3273/3273.ts
function minimumAmountOfDamageDealtToBo(root: TreeNode | null): number {
    if (!root) return 0;
    return 1 + Math.max(minimumAmountOfDamageDealtToBo(root.left), minimumAmountOfDamageDealtToBo(root.right));
}

File: 3273/3273.go
func minimumAmountOfDamageDealtToBo(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := minimumAmountOfDamageDealtToBo(root.Left)
	r := minimumAmountOfDamageDealtToBo(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}